packages/editor/src/core/plugins/markdown-clipboard.ts

```
@@ -0,0 +1,80 @@
+          }
+
+          return currentParent;
+        };
+
+        if (slice.content.childCount > 1) {
+          return markdownSerializer.serialize(slice.content);
+        } else {
+          const targetNode = traverseToParentOfLeaf(slice.content.firstChild, slice.content, slice.openStart);
+
+          let currentNode = targetNode;
+          while (currentNode && currentNode.content && currentNode.childCount === 1 && currentNode.firstChild) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)